clients/cloud/README.md

```diff
@@ -13,7 +13,7 @@ If you want to run the applications against another Kafka cluster, provide the a
 | [![](images/groovy.png)](groovy/) | [![](images/kotlin.png)](kotlin/)               | [![](images/nodejs.png)](nodejs/) |
 | [![](images/kafkacat.jpg)](kafkacat/) | [![](images/dotnet.png)](csharp/)           | [![](images/c.png)](c/) |
 | [![](images/kafka-connect-datagen.png)](kafka-connect-datagen/) | [![](images/ksql-datagen.png)](ksql-datagen/) | [![](images/rust.png)](rust/) |
-| [![](images/kafka.png)](kafka-commands/) | [![](images/clojure.png)](clojure/) | |
+| [![](images/kafka.png)](kafka-commands/) | [![](images/clojure.png)](clojure/) | [![](images/springboot.png)](java-springboot/) |
```

Comment:
Please change `springboot.png` to be an icon of just Spring (remove SR and Avro).

Reply:
done